page-index/page-index.js

```diff
@@ -552,6 +552,10 @@ const pageIndex = [
                 Title: "How to contribute",
                 type: LINK,
                 href: "https://github.com/timescale/timescaledb/blob/master/CONTRIBUTING.md"
+            }, {
+                Title: "Contribute to docs",
+                type: PAGE,
+                href: "contribute-to-docs"
```

Comment:
Maybe we should delete the "how to contribute" link to github and just point to yours?

Reply:
Note, what I wrote was specifically how to contribute to docs. Not sure if you want to expand it to include contributions to the main codebase as well. I'd need help finishing that :-)

Reply:
Ah, my bad.  Maybe then just update the above for consistency?

"Contribute to code", "Contribute to docs" ?

Reply:
Done!